stable/kong/Chart.yaml

```diff
@@ -10,5 +10,5 @@ maintainers:
 name: kong
 sources:
 - https://github.com/Kong/kong
-version: 0.5.0
-appVersion: 0.14.1
+version: 0.5.1
+appVersion: 0.14.2
```

Comment:
Wait, why is the `appVersion` changed?
Only the version of the chart should be bumped up!

Reply:
Right, totally overlooked that. @dkozlov would you please revert appVersion back `0.14.2`

Reply:
appVersion was changed to 0.14.1

Reply:
Do I need to change chart version to 0.5.2?

Reply:
Yes, as https://github.com/helm/charts/blob/master/stable/kong/Chart.yaml#L13